internal/dcache/cluster_manager/cluster_manager.go

```diff
@@ -3100,6 +3133,12 @@ func (cmi *ClusterManager) batchUpdateComponentRVState(msgBatch []*dcache.Compon
 			continue
 		}
 
+		//
+		// Map to track the RV/MV combinations that we have already seen in this batch.
+		// Key for this map is "mvName+rvName", value is the new RV state.
+		//
```

Comment:
add this

```
               // See getNextComponentRVUpdateBatch() for how it can add duplicate updates for the same
               // RV/MV combination to the batch.

```